modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2377,7 +2378,18 @@ def _arrow_type_to_dtype(cls, arrow_type):
         object
             Any dtype compatible with pandas.
         """
-        res = arrow_type.to_pandas_dtype()
+        import pyarrow
```

Comment:
Should be placed at the top of this file.

Reply:
We don't want to require `pyarrow` to be installed all the time, we need it only if a user wants to build a dataframe from `pyarrow.Table`

Reply:
makes sense